app/models/forms/server_form.py

```diff
@@ -7,6 +7,5 @@ class ServerForm(FlaskForm):
     name = StringField(validators=[InputRequired(), Length(min=3, max=80)], render_kw={"placeholder": gettext("Minecraft Server")})
     image = URLField(validators=[InputRequired(), Length(min=3, max=80)], render_kw={"placeholder": gettext("https://image.com/ssh.png")})
```

Comment:
Here you mark image input as required, but in dashboard.html.j2 you check if it's null to do something, wtf?